admin/includes/modules/search_box.php

```diff
@@ -35,6 +35,24 @@
     </div>
   </div>
     <?php
+    if (file_exists($searchBoxJs)) {
+        ?>
+        <div class="row">
+            <div class="form-group col-md-4" id="searchTermRepopulate">
+                <div class="col-sm-9 control-label"><?= TEXT_INFO_SEARCH_FILTER_REPOPULATE; ?></div>
+                <div class="col-sm-1">
+                    <?= zen_draw_checkbox_field('repopulateSearch', '', (!empty($_GET['repopulateSearch']) && $_GET['repopulateSearch'] === 'on'), '', ' id="repopulateSearch"'); ?>
+                </div>
+            </div>
```

Comment:
In order for the checkbox-label to also participate in the toggle of the checkbox, the base HTML structure needs to be similar to
```
        <div class="checkbox">
            <label>
                <?php echo zen_draw_checkbox_field('repopulateSearch', '', (!empty($_GET['repopulateSearch']) && $_GET['repopulateSearch'] === 'on'), '', ' id="repopulateSearch"'); ?>
                <?php echo ' ' . TEXT_INFO_SEARCH_FILTER_REPOPULATE; ?>
            </label>
        </div>
```

Note that I've included neither a width-defining class nor the `id` attribute for that outer `<div>` as I was unclear as to your intentions.  A similar change should be applied to the second checkbox as well.